pyproject.toml

```diff
@@ -30,10 +30,15 @@ classifiers = [
 dynamic = ["version", "readme"]
 
 dependencies = [
-    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
     "aiohttp >=3.9.2, <4.0.0",
-    "wrapt >=1.10.10, <2.0.0",
     "aioitertools >=0.5.1, <1.0.0",
+    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
```

Comment:
By the way, you should be able to write a simple check built into the pre-commit framework, verifying that these values are the same. Here's an example of embedding a few lines of python there: https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/648f815/.pre-commit-config.yaml#L45-L74.

Reply:
Nice suggestion, but I'm afraid it would need to be a little more advanced to be useful. We would need to correlate our dependency specification for `botocore` (typically a range!) with all published install requirements from the `boto3` and `awscli` ranges (!) we support. Also note that the versioning of `botocore` and `boto3` was unsynced until recently and awscli is publishing version 2 releases in parallel.

Reply:
@jakob-keller I'd just compare two strings within `pyproject.toml` and fail if they differ. If a contributor attempts to change one, they'll be forced to sync the other.

Reply:
Just to be sure we're on the same page - here's a consistent commit: https://github.com/aio-libs/aiobotocore/pull/1240/commits/972c3a6c2a2251089b905833fcb0ea5822ebd707

Reply:
Or are you suggesting to merely check that those three lines are always touched at the same time? That would work and be an improvement, but still require manual checks for consistency.

Reply:
Let's move this over to #1252